File: src/main/java/com/back/together02be/stock/dto/response/StockPriceRes.java
package com.back.together02be.stock.dto.response;

import com.back.together02be.stock.entity.Stock;

public record StockPriceRes(
	Long stockId,
	String stockCode,
	String stockName
) {
	public static StockPriceRes from(Stock stock) {
		return new StockPriceRes(
			stock.getId(),
			stock.getStockCode(),
			stock.getStockName()
		);
	}
}

File: src/main/kotlin/com/back/together02be/stock/dto/response/StockPriceRes.kt
package com.back.together02be.stock.dto.response

import com.back.together02be.stock.entity.Stock

data class StockPriceRes(
	val stockId: Long,
	val stockCode: String,
	val stockName: String
) {
    companion object {
		fun from(stock: Stock): StockPriceRes {
            return StockPriceRes(
                stock.id,
                stock.stockCode,
                stock.stockName
            )
        }
    }
}
